_data/ru/stage3.yml

```
@@ -120,22 +120,22 @@
       }
     }
   presented:
-    - date: January&#xA0;2021
+    - date: Январь&#xA0;2021
       url: >-
         https://github.com/tc39/notes/blob/master/meetings/2021-01/jan-27.md#ergonomic-brand-checks
   has_specification: true
-  description: EcmaScript proposal to provide brand checks without exceptions.
+  description: Предложение EcmaScript предоставить проверку бренда без исключений.
   authors:
     - Jordan Harband
   champions:
     - Jordan Harband
 - id: proposal-class-static-block
-  title: Class Static Block
+  title: Статический блок класса
   authors:
     - Ron Buckton
   champions:
     - Ron Buckton
-  description: Class static blocks provide a mechanism to perform additional static initialization during class definition evaluation. This is not intended as a replacement for public fields, as they provide useful information for static analysis tools and are a valid target for decorators. Rather, this is intended to augment existing use cases and enable new use cases not currently handled by that proposal.
+  description: Статические блоки класса предоставляют механизм для выполнения дополнительной статической инициализации во время оценки определения класса. Это не предназначено для замены публичных полей, поскольку они предоставляют полезную информацию для инструментов статического анализа и являются допустимой целью для декораторов. Скорее, это предназначено для расширения существующих вариантов использования и включения новых вариантов использования, которые в настоящее время не обрабатываются этим предложением. 
```

```suggestion
  description: Статические блоки класса предоставляют механизм для выполнения дополнительной статической инициализации во время исполнения определения класса. Статические блоки не предназначены для замены публичных полей, поскольку те предоставляют полезную информацию для инструментов статического анализа и являются допустимой целью для декораторов. Блоки добавлены расширения существующих и добавления новых вариантов использования, которые возможно добавятся в будущем. 
```

---

(by "that proposal" they mean https://github.com/tc39/proposal-class-fields I believe, so it's not about some new proposals in future)

```suggestion
  description: Статические блоки класса предоставляют механизм для дополнительной статической инициализации во время исполнения определения класса. Статические блоки не заменяют публичные поля, поскольку те совместимы с декораторами и могут быть полезны для инструментов статического анализа. Они скорее дополняют существующий синтаксис, а также дают новые варианты использования, не предусмотренные полями класса. 
```
, something like that